bin/mocha

```diff
@@ -45,13 +50,12 @@ const disableTimeouts = value => {
  * @ignore
  */
 const trimV8Option = value =>
-  value !== 'v8-options' && /^v8-/.test(value) ? value.slice(2) : value;
+  value !== 'v8-options' && /^v8-/.test(value) ? value.slice(3) : value;
 
 // sort options into "node" and "mocha" buckets
 Object.keys(opts).forEach(opt => {
```

Comment:
just noted, there is a Mocha flag `-r` and also one in `nodeEnv` . In this case flag is put into mocha bucket. Could lead to conflicts?

Reply:
Mocha's `--require` and `-r` are both preferred over Node's, see `isNodeFlag` in `lib/cli/node-flags.js`